p2p/host/resource-manager/limit_defaults.go

```diff
@@ -255,54 +255,50 @@ func (cfg *LimitConfig) Apply(c LimitConfig) {
 
 // Scale scales up a limit configuration.
 // memory is the amount of memory that the stack is allowed to consume,
-// for a full it's recommended to use 1/8 of the installed system memory.
+// for a dedicated node it's recommended to use 1/8 of the installed system memory.
 // If memory is smaller than 128 MB, the base configuration will be used.
 func (cfg *ScalingLimitConfig) Scale(memory int64, numFD int) LimitConfig {
-	var scaleFactor int
```

Comment:
Can we move that code change to a separate PR? Otherwise this will be very confusing if we ever have to go back in the commit history.

Reply:
Done https://github.com/libp2p/go-libp2p/pull/1685